lib/event_controller.rb

```diff
@@ -3,11 +3,17 @@
 require 'message_handler'
 require 'sinatra/base'
 require 'slack-ruby-client'
+require 'data_mapper'
+require 'dm-core'
 
 class EventController < Sinatra::Base
   attr_reader :message_handler
   def initialize()
     @message_handler = MessageHandler.new
+
+
+    DataMapper::setup(:default, ENV['DATABASE_URL'] || "sqlite3://#{Dir.pwd}/db_name.db")
```

Comment:
Hmm... Is the controller the right place to initialize the DataMapper? I feel like there should be a `main` method or similar somewhere that takes care of setting up the dependencies